packages/components/src/VirtualizedList/HeaderCheckbox/HeaderCheckbox.component.js

```diff
@@ -3,30 +3,31 @@ import React, { useMemo } from 'react';
 import classnames from 'classnames';
 import Checkbox from '../../Checkbox';
 import theme from './HeaderCheckbox.module.scss';
-import getDefaultT from '../../translate';
+import { useTranslation } from 'react-i18next';
+import I18N_DOMAIN_COMPONENTS from '../../constants';
 
 /**
  * Header renderer that displays a "Select All" checkbox on header
  */
-function HeaderCheckbox({ columnData, t }) {
+function HeaderCheckbox({ columnData }) {
 	if (!columnData.onToggleAll) {
 		return null;
 	}
 
+	const { t } = useTranslation(I18N_DOMAIN_COMPONENTS);
```

Comment:
Please check the error below
Hooks are supposed to be the first instructions of a function component (aside from assignations)

Reply:
it is the case for useMemo too